src/AElf.Kernel.CodeCheck/Application/ICodeCheckProposalService.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace AElf.Kernel.CodeCheck.Application;
+
+public interface ICodeCheckProposalService
+{
+    void AddToReleasedProposal(Hash proposalId, Hash proposalInputHash, long height);
```

Comment:
The naming doesn't follow English grammar. Maybe change to `AddReleasableProposal`.

Reply:
Got it.